src/ai/backend/manager/models/domain.py

```diff
@@ -196,7 +201,9 @@ async def mutate(cls, root, info, name, props):
 
 
 class DeleteDomain(graphene.Mutation):
-
+    '''
```

Comment:
For new codes, let's use double quotes for docstrings as PEP-8 and PEP-257 suggests.

Reply:
I replaced single quotes to double quotes.